.github/workflows/scenarios-permian.yml

```diff
@@ -0,0 +1,179 @@
+
+      # Configure location of installation repositories for the scenario
+      # Also default boot.iso is defined by the value of urls.installation_tree
+      # of kstestParams event structure.
+      - name: Set installation_tree for the scenario
+        id: set_installation_urls
+        run: |
+          set -eux
+          if [ "${{ matrix.scenario }}" == "rhel8" ]; then
+            echo "::set-output name=installation_tree::http://download.eng.bos.redhat.com/rhel-8/development/RHEL-8/latest-RHEL-8/compose/BaseOS/x86_64/os"
+            echo "::set-output name=modular_url::http://download.eng.bos.redhat.com/rhel-8/development/RHEL-8/latest-RHEL-8/compose/AppStream/x86_64/os"
+          elif [ "${{ matrix.scenario }}" == "rhel9" ]; then
+            echo "::set-output name=installation_tree::http://download.eng.bos.redhat.com/rhel-9/development/RHEL-9/latest-RHEL-9/compose/BaseOS/x86_64/os"
+            echo "::set-output name=modular_url::http://download.eng.bos.redhat.com/rhel-9/development/RHEL-9/latest-RHEL-9/compose/AppStream/x86_64/os"
+          else
+            echo "::set-output name=installation_tree::"
+            echo "::set-output name=modular_url::"
+            echo "Installation tree location for ${{ matrix.scenario }} not configured"
+          fi
+
+      - name: Run scenario ${{ matrix.scenario }} in container
+        working-directory: ./permian
+        run: |
+          sudo --preserve-env=TEST_JOBS \
+          PYTHONPATH=${PYTHONPATH:-}:${{ github.workspace }}/tclib \
+          ./pipeline --debug-log permian.log \
+            -o library.directPath="${{ github.workspace }}/kickstart-tests/testlib" \
+            -o workflows.dry_run=False \
+            -o kickstart_test.timeout=${LAUNCHER_TIMEOUT_MINUTES}m \
+            -o kickstart_test.retry_on_failure=True \
+            -o kickstart_test.kstest_local_repo="${{ github.workspace }}/kickstart-tests" \
+            run_event '{"type":"github.scheduled.daily.kstest.${{ matrix.scenario }}","bootIso":{"x86_64":"${{ steps.boot_iso_from_scenario.outputs.boot_iso }}"},"kstestParams":{"platform":"${{ steps.platform_from_scenario.outputs.platform }}","urls":{"x86_64":{"installation_tree":"${{ steps.set_installation_urls.outputs.installation_tree }}","modular_url":"${{ steps.set_installation_urls.outputs.modular_url }}"}}}}'
+
+      - name: Collect anaconda logs
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: 'logs-${{ matrix.scenario }}'
+          # skip the /anaconda subdirectories, too large
+          path: |
+            kickstart-tests/data/logs/kstest.log
+            kickstart-tests/data/logs/kstest-*/*.log
+            kickstart-tests/data/logs/kstest-*/anaconda/lorax-packages.log
+
+      - name: Collect Permian logs
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: 'logs-permian-${{ matrix.scenario }}'
+          path: |
+            permian/permian.log
```

Comment:
Does setting the xunit Permian output leave any "normal" logs?

Reply:
This log is created by the permian run (`--debug-log permian.log`).
AFAIK xunit reporter creates only .xml logs.